run/GCHP/GCHP.rc.template

```diff
@@ -37,7 +37,7 @@ DYNAMICS_DT: 600
 SOLARAvrg: 0
 IRRADAvrg: 0
 
-GCHPchem_REFERENCE_TIME: 121000
+GCHPchem_REFERENCE_TIME: 001000
```

Comment:
Why is this changing?

Reply:
I have removed these and the other changes to common run files for now.